java/ql/lib/change-notes/2022-08-25-taint-model-spring-crudrepository.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added taint model for `org.springframework.data.repository.CrudRepository.save()`. This change yields an additional true positive SQL Injection on WebGoat.
```

Comment:
I don't think it's necessary to mention the WebGoat result in the change note.

Otherwise, and aside from the CodeQL alert above, LGTM.

```suggestion
* Added taint model for `org.springframework.data.repository.CrudRepository.save()`.
```

Reply:
done.